BattleNetwork/netplay/bnPacketShipper.h

```diff
@@ -10,6 +10,8 @@
 #include "../bnNetManager.h"
 #include "bnPacketAssembler.h"
 
+constexpr uint16_t DEFAULT_MAX_PAYLOAD_SIZE = 1300;
```

Comment:
note to self: we may want to communicate this as part of the BigData protocol so clients can agree on space requirements.

Reply:
I will add an input argument for this